### DISCUSSION TITLE
Add abilty to edit skin names

### DISCUSSION THREAD

**Discussion by chicco-carone** [ppy/osu#31214] [Ideas] [answered]
In stable it was possible to edit a skin's name by just editing the folder name but due to the way that lazer stores data its impossible to change the name of a skin, it would need to be edited in the db which is something that the user can't (to my knowledge) edit manually so a button in the skin editor like "save as" to maybe copy it and save it with a new name or just a rename button would be a great addition. In my experience the skin dropdown is way too disorganized since you have first the built in osu skins which have the name formatted decently and then any custom one has way too many spaces an weird symbols which makes them harder to find.

**Comment by Joehuu** [comment, marked as answer]
May be implemented in https://github.com/ppy/osu/pull/31220.
